subiquity/models/filesystem.py

```diff
@@ -976,14 +1110,77 @@ def _probe_bootloader(self):
 
     def __init__(self):
         self.bootloader = self._probe_bootloader()
-        self._disk_info = []
+        self._probe_data = None
         self.reset()
 
     def reset(self):
-        self._actions = [
-            Disk.from_info(self, info) for info in self._disk_info]
+        if self._probe_data is not None:
+            config = storage_config.extract_storage_config(self._probe_data)
+            self._actions = self._actions_from_config(
+                config["storage"]["config"],
+                self._probe_data['blockdev'])
+        else:
+            self._actions = []
         self.grub_install_device = None
 
+    def _actions_from_config(self, config, blockdevs):
```

Comment:
This could use a comment on how config entries are mapped to actions and what exclusions imply.

Reply:
Added.